cmd/up.go

```diff
@@ -442,3 +447,28 @@ func printDisplayContext(message, namespace, name string, ports []model.Forward)
 	}
 	fmt.Println()
 }
+
+func checkWatchesConfiguration() {
+	if runtime.GOOS != "linux" {
+		return
+	}
+
+	w := "/proc/sys/fs/inotify/max_user_watches"
+	f, err := ioutil.ReadFile(w)
+	if err != nil {
+		log.Errorf("Fail to read %s: %s", w, err)
+	}
+
+	c, err := strconv.Atoi(string(f))
+	if err != nil {
+		log.Errorf("Fail to parse the value of  max_user_watches: %s", err)
+	}
+
+	log.Debugf("max_user_watches = %d", c)
+
+	if c <= 8192 {
+		log.Yellow("The value of /proc/sys/fs/inotify/max_user_watches is too low. This can affect Okteto's file synchronization performance.")
+		log.Yellow("We recommend you to raise it to at least 524288 to ensure proper performance.")
```

Comment:
@pchico83 do you have the value you used with the linux users? I put this based on the recommendation for node I saw online.